media/inf/scsi.inf

```diff
@@ -100,6 +100,7 @@ ReactOS = "Equipo de ReactOS"
 SCSIClassName = "Controladoras SCSI y RAID"
 
 GenericMfg = "(Controladoras estándar SCSI y RAID)"
+GEN_SCSIADAPTER.DeviceDesc = "Controlador Huésped de SCSI/RAID"
 PCI\VEN_104B&CC_0100.DeviceDesc = "Controladora SCSI BusLogic"
```

Comment:
"controlador" vs. "controladora" ?

Reply:
It depends. The better is to use the male "controlador", as Windows did. But both can be possible. I prefer the masculine. I will fix this and every "controladora" appears on the code, thanks for your puntualization.